internal/pkg/template/diff/formatter.go

```diff
@@ -30,7 +30,14 @@ func (f *seqItemFormatter) formatYAML(node *yaml.Node) ([]byte, error) {
 }
 
 func (f *seqItemFormatter) formatMod(node diffNode) string {
-	return fmt.Sprintf("- %s -> %s", node.oldYAML().Value, node.newYAML().Value)
+	oldValue, newValue := node.oldYAML().Value, node.newYAML().Value
+	if v, err := yaml.Marshal(node.oldYAML()); err == nil { // NOTE: Marshal handles YAML tags such as `!Ref` and `!Sub`.
+		oldValue = strings.TrimSuffix(string(v), "\n")
```

Comment:
Realistically, I think it's unlikely that this will error out, because:
1. The yaml node was unmarshalled from a working YAML file
2. The yaml node isn't modified

If for whatever reason there is indeed a marshal error, I thought `node.oldYAML().Value` should already be enough for users to pinpoint the location of the change, in most cases. Therefore, surfacing `node.oldYAML().Value` might be moroe helpful than surfacing the error (again, if there is any 😂 ) in this case 🤔